src/autotuner/genetic_search.cc

```diff
@@ -72,9 +74,35 @@ void mutate(
   }
 }
 
-void normalizeVector(std::vector<double>& v) {
+double mean(std::vector<double>& v) {
+  if (v.empty()) {
+    throw std::invalid_argument("Cannot compute the mean of an empty vector.");
+  }
   auto sum = std::accumulate(v.begin(), v.end(), 0.0);
+  return sum / v.size();
+}
+
+double stdv(std::vector<double>& v, double mean) {
```

Comment:
can these be moved to the utils file https://github.com/facebookresearch/TensorComprehensions/blob/master/src/autotuner/utils/utils.cc

where we already have mathematical functions implemented?